optuna/integration/skopt.py

```diff
@@ -266,6 +274,8 @@ def _complete_trial_to_skopt_observation(self, study, trial):
                 param_value = (param_value - distribution.low) // distribution.q
             if isinstance(distribution, distributions.IntUniformDistribution):
                 param_value = (param_value - distribution.low) // distribution.step
+            if isinstance(distribution, distributions.IntLogUniformDistribution):
+                param_value = np.log(param_value)
```

Comment:
Actually, this causes an error when `np.log` takes `param_value=1` in the following example.

```python
import optuna

def objective(trial):
    x = trial.suggest_uniform('x', -10, 10)
    y = trial.suggest_int('y', 1, 9, 2, True)
    return x**2 + y

sampler = optuna.integration.SkoptSampler()
study = optuna.create_study(sampler=sampler)
study.optimize(objective, n_trials=10)
```

Reply:
Oops,  these lines are unnecessary.